stack/tasks/create-a-m-o.js

```diff
@@ -168,10 +360,16 @@ const writeIndexScss = (path, _name) => {
 };
 
 const writePreviewAndHtml = (path, _name) => {
+  const markup = isBuiltin ?
+    outdent`<${buildinElement} is="axa-${_name}" classes="${element}-${_name}"></${buildinElement}>
+    `
+    :
+    outdent`<axa-${_name} classes="${element}-${_name}"></axa-${_name}>
+    `;
+
   fs.writeFileSync(
     `${path}/_preview.html`,
-    outdent`<axa-${_name} classes="${element}-${_name}"></axa-${_name}>
-    `
+    markup
     , handleError,
   );
   fs.writeFileSync(
```

Comment:
article i9in built ins maybe should be remobved

Reply:
and also display block on the scss file

Reply:
Okay, `<article>` only used for non-built-in organisms, else `<div>`???